redirects.yaml

```diff
@@ -460,8 +460,8 @@ foundation-cloud-build/?: "/openstack/consulting"
 frs/?: "/"
 gateways/?:  "/internet-of-things/gateways"
 get/?: "/download"
-getubuntu/?: "/download"
-GetUbuntu/?: "/download"
+getubuntu.*/?: "/download"
```

Comment:
it seems we are still getting a bit of  traffic on pages that don't 404

eg. ` /getubuntu/releasenotes? -> https://wiki.ubuntu.com/Releases`

Reply:
Can you redirect selected URLs from the 404s to save redirecting URLs that should 404 like /getubuntu/404 and not redirect existing paths.

Reply:
These comments seem to be addressed, it's redirecting all to `/download`